### ISSUE TITLE
Error - Addon local_airsend does not exist

### ISSUE THREAD

**Issue by NicolasQuenon** [devmel/hass_airsend#7]
Hi ! 

I follow all your process  but I have an errror : 
`Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 253, in _async_setup_component
    result = await task
  File "/config/custom_components/airsend/__init__.py", line 25, in async_setup
    addon_info: dict = await async_get_addon_info(hass, "local_airsend")
  File "/usr/src/homeassistant/homeassistant/components/hassio/handler.py", line 64, in async_get_addon_info
    return await hassio.get_addon_info(slug)
  File "/usr/src/homeassistant/homeassistant/components/hassio/handler.py", line 51, in _wrapper
    raise HassioAPIError(data["message"])
homeassistant.components.hassio.handler.HassioAPIError: Addon local_airsend does not exist`

I put this configuration  : 
`airsend:
  devices:
    Volet salon:
      id: 25591
      type: 4098
      spurl: !secret spurl
      channel:
         id: 26848
         source: 2076914944`

And my secret on secret.yalm : 
`spurl: sp://MyPASSWORD@192.168.1.22`

I have your plugin in Hacs, and I add AirSend Web service in Home Assistant Add-on.

May you help me do solve this problem ? 

Thanks

**Comment by emilmansson86** [comment]
I have extactly the same problem. Have you found any solution?

**Comment by titou4307** [comment]
Same problem

**Comment by laugel** [comment]
Same problem here. 
I don't know what is the cause exactly, but reading at the code, I found a workaround : add the `internal_url` key (and its value) in the `airsend` section of your `configuration.yaml`. 
Such as :
![image](https://github.com/devmel/hass_airsend/assets/5048661/1ad72be9-1a7c-4722-b4d1-236c9088e555)

**Comment by falconslag** [comment]
Hi,

I am using HA OS in a VM. I've installed the Module for local usage. I don't have that problème.
But the integration was not deployed via HACS. I used the command line using wget. It s not supposed to have any side effect.

I've also tried for a while with a 'home made' docker container for the Airsend module to work with HA also in a container.

The code is supposed to retrieved the name of the module automatically for the URL to call.
It is also displayed on the module information tab.
![Capture d’écran 2023-07-27 184748](https://github.com/devmel/hass_airsend/assets/86597737/f08ad800-9731-4058-ba6e-939ba26fad15)

in the screenshot above, mine is currently f5e830cd-airsend.

I've locally modified the code to call a specific IP address (the one of my home made container). It worked.

For that, I modified  "custom_components\airsend\cover.py" file and forced the addons_url to "http://192.168.x.x:33863/".
I commented lines 25 and 26 and added the new line after, as below
\#        addon_info: dict = await async_get_addon_info(hass, 'local_airsend')
\#        ip = addon_info["ip_address"]
        addons_url = "http://192.168.x.x:33863/"

The integration then calls the Airsend locally running in a dedicated container, outisde of Home Assistant container.

Regards,
Falcon

**Comment by falconslag** [comment]
Last info, but I am going to make it confirmed by Devmel support  because I am not sure, but I remember one guy from their support telling me that some protocoles are only available via the Cloud Plateform.

For exemple, I think Velux IO is one of them. I say that because my Velux were repaired 2 days ago, and when I tried to manage them using local module, nothing happened. I switched back to cloud and the API key and it worked. After, I may have kept a mistake somewhere in the config file.

Unfortunatly, I don't have time in the coming weeks to do more test. So a ticket to Devmel support will be more efficient.

I'll keep you posted.

@+
Falcon
